setup.py

```diff
@@ -0,0 +1,51 @@
+import os
+import subprocess
+import multiprocessing
+from distutils.core import setup
+from distutils.errors import LibError
+from distutils.command.build import build as _build
+
+class build(_build):
+    @staticmethod
+    def _configure():
+        if subprocess.call(['./configure']) != 0:
```

Comment:
Shouldn't this do `['./configure', '--prefix', sys.prefix]` or something similar?